src/main/java/sirius/biz/jdbc/tenants/Tenants.java

```diff
@@ -147,10 +148,12 @@ public void assertTenantOrParentTenant(TenantAware tenantAware) {
             return;
         }
 
-        if (!Objects.equals(tenantAware.getTenantAsString(), getCurrentTenant().map(Tenant::getUniqueName).orElse(null))
```

Comment:
couldn't / shouldn't getTenantAsString return the uniqueName instead? As we use it everywhere in the TenantUserManager